proxy/src/main/java/com/velocitypowered/proxy/VelocityServer.java

```diff
@@ -209,9 +209,8 @@ void start() {
 
     // Initialize commands first
     commandManager.register("velocity", new VelocityCommand(this));
-    commandManager.register("server", new ServerCommand(this));
-    commandManager.register("shutdown", ShutdownCommand.command(this),
-        "end", "stop");
+    commandManager.register("goto", new ServerCommand(this));
```

Comment:
Changing the `server `command to `goto` makes no sense, plus it breaks many setups of several servers, and it seems that it could only be useful for your individual setup